payment_payzen/data/mail_templates_data.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <record id="sale_mail_template_payzen_info" model="mail.template">
+        <field name="name">Sale Order - Send By Mail (Payzen info)</field>
+        <field name="model_id" ref="model_sale_order"/>
+        <field name="body_html" type="html">
+            <t t-set="access_action" t-value="object._get_access_action(force_website=True)"/>
+            <t t-set="is_online" t-value="access_action and access_action['type'] == 'ir.actions.act_url'"/>
+            <t t-set="doc_name"
+               t-value="'quotation' if object.state in ('draft', 'sent') else 'order confirmation'"/>
+            <t t-set="share_url" t-value="object.get_mail_url()"/>
+            <t t-set="base_url" t-value="object.get_base_url()"/>
+            <t t-set="access_url" t-value="is_online and share_url and base_url + share_url or ''"/>
+            <p>
+                Cher <t t-out="object.partner_id.name"/>
+                <t t-if="object.partner_id.parent_id">
+                    (<i t-out="object.partner_id.parent_id.name"/>)
+                </t>
+                ,
+            </p>
+            <p>
+                Vous trouverez ci-dessous les détails de
+                <t t-if="ctx.get('proforma')">votre facture pro-forma</t>
+                <t t-else="">votre commande <strong t-out="object.name"/></t>
+                pour un montant total de
+                <strong t-out="format_amount(object.amount_total, object.currency_id)"/>
+                <t t-if="object.recurring_payment_months &lt;= 1">
+                    le
+                    <t t-out="format_date(object.date_order)"/>
+                </t>
+                auprès de<t t-out="object.company_id.name"/>.
+            </p>
+
+            <p t-if="object.recurring_payment_months > 1">
+                <t t-if="object.recurring_payment_months">
+                    Payable en <t t-out="object.recurring_payment_months"/> fois
+                    <br/>
+                </t>
+                <t t-if="object.recurring_first_payment_amount">
+                    Premier paiement de
+                    <t t-out="format_amount(object.recurring_first_payment_amount, object.currency_id)"/>
+                    le
+                    <t t-out="format_date(object.date_order)"/>
+                </t>
+                <t t-else="">
+                    Avec des paiements à hauteur de
+                    <t t-out="format_amount((object.recurring_payment_monthly_amount), object.currency_id)"/>
```

Comment:
Just being picky, the secon `()` for the field are not needed. Same in other ocurrencies
```suggestion
                    <t t-out="format_amount(object.recurring_payment_monthly_amount, object.currency_id)"/>
```

Reply:
Yep, likely a leftover from auto-converted qweb (I copied this from the db vs. the original jinja one in the repo).
Will fix.